helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -240,6 +240,12 @@ controller:
     # Do not overwrite this value.
     enabled: false
 
+    # controller.service.externalTrafficPolicy
+    # Configures kube-proxy, denotes if this Service desires to route external traffic to node-local or cluster-wide endpoints
```

Comment:
```suggestion
    # Configures kube-proxy, denotes if this Service desires routing external traffic to node-local or cluster-wide endpoints
```

Minor but this flows better for me.

Reply:
How about
```suggestion
    # Configures kube-proxy, denotes if this Service desires to have external traffic routed to node-local or cluster-wide endpoints
```

Btw that sentence was initially taken from official k8s docs :)

Reply:
Nice I like that more. Also funny it came from upstream. Maybe it's a 🇬🇧 vs 🇺🇸 english thing :)